samples/dashboards/cypress/component-datagrid/demo.mjs

```diff
@@ -82,7 +83,8 @@ Dashboards.board('container', {
             sync: {
                 highlight: true,
                 extremes: true
-            }
+            },
+            columnsToShow: ['Food', 'Vitamin A']
```

Comment:
IMO `visibleColumns` or `displayColumns` sound more intuitive.

Reply:
Similar suggestion to `notVisible` param, IMO `hidden` sounds more user-friendly.